src/Sandbox.MqttHive/BeeClient.cs

```diff
@@ -0,0 +1,78 @@
+﻿using HiveMQtt.Client;
+using HiveMQtt.Client.Events;
+using HiveMQtt.Client.Options;
+using HiveMQtt.MQTT5.Types;
+using Sandbox.MqttHive;
+using System.Diagnostics;
+
+namespace Sandbox.MqttBee;
+
+public class BeeClient : IBee
+{
+    public string ID { get; }
+    public bool IsConnected => _client.IsConnected();
+    public IList<TopicFilter> ScopedTopics { get; }
+
+    public BeeClient(HiveMQClientOptions options)
+    {
+        ID = options.ClientId ?? Guid.NewGuid()
+                                     .ToString();
+
+        _client = new HiveMQClient(options);
+
+        ScopedTopics = new List<TopicFilter>()
+        {
+            new(Topics.HiveRoot),
+            new(Topics.RootTopic(this)),
+            new(Topics.CommandTopic(this)),
+            new(Topics.DataTopic(this))
+        };
+    }
+
+    public async Task Initialise()
```

Comment:
It is standard practice to use US english in code 
```suggestion
    public async Task Initialize()
```

Reply:
Boo